config.py

```diff
@@ -159,19 +159,31 @@ class Config:
             "offline_warning_enabled": True,
             "publish_benchmark_results": True,
             "max_builds": 1,
-            "build_timeout": 150,
+            "build_timeout": 120,
```

Comment:
unrelated change: due to https://github.com/apache/arrow/issues/40775 being fixed we can lower the timeout for the thinkcentre